### ISSUE TITLE
Debug Side Bar: VARIABLES is empty when no active threads

### ISSUE THREAD

**Issue by SAbboushi** [zero-plusplus/vscode-autohotkey-debug#7]
```
#SingleInstance Force 	; Allow only one running instance of script
#Persistent 			; Keep the script permanently running until terminated

x := 1
MsgBox 12
y := 2
```
You have told me how I can get VARIABLES pane to show me current values when MsgBox is displayed by clicking Pause button.

I hoped this would also work after #Persistent script has finished running, but it does not: VARIABLES pane remains empty:
![image](https://user-images.githubusercontent.com/1964198/84031883-15707b80-a95c-11ea-91c6-b35f8f252402.png)

**Comment by zero-plusplus** [comment]
That is normal behavior.

Try with SciTE4AutoHotkey. In my case, the following message was displayed.
> Script pausing is not supported in this AutoHotkey version!

Technically speaking, AutoHotkey goes into a wait state when all calls are finished (that is, the stack frame is empty).
You pressed the pause button while in this state.
However, information such as variables is obtained through the stack frame, so you cannot see that information.

**Comment by SAbboushi** [comment]
> That is normal behavior.
> 
> Try with SciTE4AutoHotkey. In my case, the following message was displayed.
> > Script pausing is not supported in this AutoHotkey version!
> 

It works for me in SciTE4AutoHotkey: when I click "pause" button, all values are refreshed in Variables list

This is less of a problem for scripts that cannot create another thread (like simple script above), but I think is a big problem for scripts that use hotkeys and timers where I want to see state when last thread finished before e.g. I press a hotkey that starts a new thread

**Comment by zero-plusplus** [comment]
I just checked but no error message was displayed
I debugged exactly the same source code, but...
You're right This is a feature that should be implemented.

I tried implementing it for the time being, but it may take some time because a strange phenomenon occurs.

The first time is displayed without problems, but from the second time onward, the variables are not displayed unless the call stack is selected.
